src/test/java/com/synopsys/integration/detect/tool/cache/InstalledToolLocatorTest.java

```diff
@@ -0,0 +1,21 @@
+package com.synopsys.integration.detect.tool.cache;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.google.gson.Gson;
+
+public class InstalledToolLocatorTest {
```

Comment:
Shouldn't there be a test for loading and retrieving data from `InstalledToolManager`?